guide/daily-spending-wallet/sending.md

```diff
@@ -198,7 +198,7 @@ On the Lightning network, payments are passed between nodes to get from the send
 
 **Lightning service provider fees**
 
-In certain situations, the Lightning wallet may not have enough channel liquidity to send a payment. Wallet providers may offer to alleviate these friction points, and earn additional fees. A common scenario is the automatic opening of a payment channel when a wallet attempts to send a payment larger than their outbound capacity.
+In certain situations, the Lightning wallet may not have enough channel [liquidity]({{ '/guide/how-it-works/liquidity/' | relative_url }}) to send a payment. [Wallet providers]({{ '/guide/how-it-works/lightning-service-providers/' | relative_url }}) may offer to alleviate these friction points, and earn additional fees. A common scenario is the automatic opening of a payment channel when a wallet attempts to send a payment larger than their outbound capacity.
```

Comment:
```suggestion
Lightning wallets may require [additional services]({{ '/guide/how-it-works/lightning-services/' | relative_url }}) to solve certain usability issues. An example being a lightning service provider (LSP) opening payment channels and providing [inbound liquidity]({{ '/guide/how-it-works/liquidity/' | relative_url }}) for the user so the can receive payments. As these services are offered by third parties, additional fees may be charged.
``` 
Little bit of a rephrase to fit more nicely in with the lightning services page.

Reply:
Needs some minor fixing in the second to last sentence - should be "... for the user, so they...".

Reply:
Updated